modules/govuk/manifests/apps/rummager/rabbitmq.pp

```diff
@@ -34,13 +34,14 @@
 
   # match everything on queue 2 while we test
   govuk_rabbitmq::consumer { 'rummager-v2':
-    ensure        => $toggled_ensure,
-    amqp_pass     => $password,
-    amqp_exchange => 'published_documents',
-    amqp_queue    => 'rummager_to_be_indexed',
-    routing_key   => '*.links',
-    amqp_queue_2  => 'rummager_govuk_index',
-    routing_key_2 => '#',
+    ensure                      => $toggled_ensure,
+    amqp_pass                   => $password,
+    amqp_exchange               => 'published_documents',
+    amqp_queue                  => 'rummager_to_be_indexed',
+    routing_key                 => '*.links',
+    extra_read_permissions      => 'govuk_index_durable|govuk_index_transient',
+    extra_write_permissions     => 'govuk_index_durable|govuk_index_transient',
+    extra_configure_permissions => 'govuk_index_durable|govuk_index_transient',
   }
```

Comment:
You no longer seem to be creating `rummager_govuk_index` not sure if this was intentional?